.github/workflows/upload.yml

```diff
@@ -0,0 +1,52 @@
+name: Upload
+on:
+  release:
+    types: [published]
+
+jobs:
+  upload:
+    runs-on: ubuntu-latest
+    env:
+      HYPOTHESIS_PROFILE: ci
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Build and install Mr Mustard
+        run: |
+          python -m pip install --upgrade pip wheel
+          python setup.py bdist_wheel
+          pip install dist/mrmustard*.whl
+
+      - name: Install test dependencies
+        run: |
+          pip install -r requirements.txt
+          pip install wheel pytest pytest-cov pytest-mock hypothesis --upgrade
+
+      - name: Run tests
+        run: |
+          python -m pytest tests -p no:warnings --tb=native
+
+      - name: Publish
+        uses: pypa/gh-action-pypi-publish@master
+        with:
+          user: __token__
+          password: ${{ secrets.PIPY_TOKEN }}
+
+      - name: Uninstall existing versions and pause
+        run: |
+          python -m pip uninstall mrmustard -y
+          sleep 5m
```

Comment:
I can't believe I'm reading this line and getting jealous 😂

Reply:
lol 🤣